patterns/transform-navigation/x-app.html

```diff
@@ -109,8 +109,17 @@
           }
         }
 
-      }
+      },
+
+      observers: ['onLayoutChange(wideLayout)'],
+
+      onLayoutChange: function(wide) {
+        var drawer = this.$.drawer;
 
+        if (wide && drawer.opened) {
```

Comment:
It seems like this code is missing the the case for `!wide && !drawer.opened` is that case needed?